specification/healthbot/resource-manager/Microsoft.HealthBot/preview/2020-10-20-preview/examples/CheckNameAvailability.json

```diff
@@ -0,0 +1,18 @@
+{
+  "parameters": {
+    "api-version": "2020-10-20-preview",
```

Comment:
For healthbot, is it related with the deleted account API in this PR?
If not, please open another PR for it.